index.html

```diff
@@ -28,53 +53,142 @@
         </div>
       </div>
     </nav>
+
     <div id="content">
       <div class="container">
         <div id="carousel"></div>
-        <div class="row">
-          <div class="col-md-12">
-            <div class="stats">
-              <div class="row">
-                <div class="col-md-2"></div>
-                <div class="col-md-4">
-                  <h4><div class="glyphicon glyphicon-signal"></div> Progress:</h4>
-                  <p>5/25 words</p>
-                  <div class="progress">
-                    <div class="progress-bar progress-bar-info progress-bar-striped" role="progressbar" aria-valuenow="20" aria-valuemin="0" aria-valuemax="100" style="width: 20%;">
-                      20%
+        <div class="col-md-12">
```

Comment:
Correct, this table was made by hand in order to present the mock-up for the main page. Enrico is already working on the next version of the main page.